vtysh/vtysh_config.c

```diff
@@ -772,6 +772,7 @@ void vtysh_config_write(void)
 {
```

Comment:
I'm a bit confused why/where writing this out would be useful, need to read some code to restore understanding…

[this is not a blocking comment, maybe a question though]

Reply:
I was just following the existing examples, I guess?